File: C/977A-Wrong Subtraction.c
/*
    Solution to Codeforces problem 977A
    Copyright (c) GeneralMing. All rights reserved.

    https://github.com/GeneralMing/codeforces
*/

#include<stdio.h>
#include<string.h>

int main()
{
    int n, k;
    scanf("%d", &n);
    scanf("%d", &k);
    while(k--)
    {
        if(n%10)
            n--;
        else
            n = n/10;
    }
    printf("%d\n", n);
    return 0;
}

File: Python/977A-Wrong Subtraction.py
"""
    Solution to Codeforces problem 977A
    Copyright (c) GeneralMing. All rights reserved.

    https://github.com/GeneralMing/codeforces
"""

n, k = input().split(" ")
n = int(n)
k = int(k)
while(k != 0):
	if(n%10):
		n -= 1
	else:
		n = int(n/10)
	k -= 1
print(n)